style.css

```diff
@@ -638,3 +321,163 @@ textarea {
   line-height: 28px;
   padding-left: 10%;
 }
+
+.hide-in-mobile {
+  display: none;
+}
+
+@media only screen and (min-width: 990px) {
+  .hide-in-desktop {
+    display: none;
+  }
+
+  .show {
+    display: block;
+  }
+
+  .navbarlist {
+    display: flex;
+  }
+
+  ul,
+  li a {
+    text-decoration: none;
+    color: #f9f9f9;
+    margin-right: 20px;
+  }
+
+  .logo > a {
+    margin-left: 15px;
+  }
+
+  .first-section {
+    background-image: url("IMG1/desktop-background.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .header-container h2 {
+    font-size: 90px;
+    line-height: 105px;
+    margin: 10%;
+    width: 100%;
+    height: 96px;
+  }
+
+  .header-container h3 {
+    font-size: 40px;
+    line-height: 47px;
+    width: 1156px;
+    height: 72px;
+    margin: 24px 0;
+    text-align: center;
+  }
+
+  .intro-container {
+    display: grid;
+    grid-template-columns: 1fr;
+    margin: 0 33%;
+  }
+
+  .top-divider {
+    width: 120%;
+  }
+
+  .intro-container p {
+    font-size: 20px;
+    line-height: 32px;
+    width: 447px;
+    height: 155px;
+    color: #b3bac5;
+  }
+
+  .lang-framework-container {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: 1fr;
+    grid-column-gap: 0;
+    grid-row-gap: 0;
+    margin-top: 4px;
+  }
+
+  .title {
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 29.33px;
+  }
+
+  .social-media-container {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: 1fr;
+    grid-column-gap: 20px;
+    grid-row-gap: 0;
+    align-items: center;
+    margin: 0 -200px;
+    justify-content: space-evenly;
+  }
+
+  .work-image1 {
+    display: flex;
+    justify-content: center;
+  }
+
+  .main-content-area {
+    margin-top: 232px;
+  }
+
+  .title-container > img {
+    margin: 30px 142px;
+    background-repeat: no-repeat;
+  }
+
+  .main-body {
+    background-image: url("IMG1/desktop-background2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .projects-container {
+    display: flex;
+    align-items: center;
+    margin: 100px;
+  }
+
+  .projects-container:nth-child(even) {
+    flex-direction: row-reverse;
+  }
+
+  .form-intro p {
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+  }
+
+  .projects-container:nth-child(odd) > p {
+    padding-right: 32px;
+  }
+
+  .image-placeholder > img {
+    width: 448px;
+  }
+
+  .content {
+    margin-left: 22px;
+  }
+
+  .form-container {
+    padding: 114px;
+  }
```

Comment:
- [ ] By giving padding to the form it's affecting the icons in the footer which means it's creating more space between the icons and the horizontal line under the form. Kindly fix it.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/155732214-d7bf1fda-0c99-4da1-9714-95935cca9c44.png)


### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/155732330-13e76166-8bf4-4646-a8ea-50555aeadfdc.png)